api.md

```diff
@@ -431,12 +434,72 @@ PostGIS geospatial indexes).  During testing, you might check your
 total chunk sizes via the [`chunk_relation_size`](#chunk_relation_size)
 function.
 
-**Space partitions**: In most cases, it is advised for users not to use
+**Space partitions:** In most cases, it is advised for users not to use
 space partitions. The rare cases in which space partitions may be useful
 are described in the [add dimension][] section.
 
 ---
 
+## create_distributed_hypertable() [](create_distributed_hypertable)
+
+A specialized implementation of [`create_hypertable`](#create_hypertable)
+designed for creating hypertables distributed across multinode 
+environments.  The required and optional arguments are the same as for
+`create_hypertable`, as is the returned value.  The main difference is
+that this command will default the `replication_factor` optional
+parameter to a value of 1.
+
+#### Sample Usage [](create_distributed_hypertable-examples)
+
+Create a table `conditions` spread across the entire distributed
+database.
+```sql
+SELECT create_distributed_hypertable('conditions', 'time');
+```
+
+Create a table `conditions` which will be partitioned across data
+nodes by the 'location' column.  Note that the number of space
+partitions is automatically equal to the number of data nodes in this
+case.
+```sql
+SELECT create_distributed_hypertable('conditions', 'time', 'location');
+```
+
+Create a table `conditions` using a specific set of data nodes.  Each
+chunk will be written to two data nodes.
+```sql
+SELECT create_distributed_hypertable('conditions', 'time', 'location', replication_factor => 2,
+    data_nodes => '{ "data_node_1", "data_node_2", "data_node_4", "data_node_7" }');
+```
+
+#### Best Practices [](create_distributed_hypertable-best-practices)
+
+**Space partitions:** As opposed to the normal [`create_hypertable` best practices]
+(#create_hypertable-best-practices), space partitions are highly
```

Comment:
You cannot have a line break between the two parts of the link here 
> [`create_hypertable` best practices] (#create_hypertable-best-practices)

Reply:
fixed